ui/src/main/webapp/app/settings/settings.html

```diff
@@ -42,6 +42,16 @@ <h3>Result count limit</h3>
                                     </div>
                                 </md-input-container>
                             </md-list-item>
+                            <md-list-item class="md-2-line">
+                                <div class="md-list-item-text" flex="60">
+                                    <h3>Clear Operation Chain</h3>
+                                    <p>Clear the operation chain after each execution</p>
+                                </div>
+                                <div flex></div>
+                                <md-input-container class="md-secondary">
+                                    <md-checkbox ng-click="ctrl.saveClearChainCheckbox()" ng-checked="ctrl.clearChainCheckbox"></md-checkbox>
```

Comment:
I think you have to add something like an ng-change to update the service when the value changes.